@navikt/core/react/src/form/search/Search.tsx

```diff
@@ -215,7 +200,6 @@ export const Search = forwardRef<HTMLInputElement, SearchProps>(
               value={value ?? internalValue}
               onChange={(e) => handleChange(e.target.value)}
               type="search"
-              role={role ?? "searchbox"}
```

Comment:
Should `role` be set to default `searchbox` in prop-destructure, or is there another reason to remove `searchbox`?

Reply:
I decided to remove role=searchbox after reading this: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/searchbox_role#examples
As I interpret it, role=search is redundant when using type=search. I might be wrong, though. I asked in #aksel-utviklere, but got no responses so far.